src/storyMap/components/UserStoryMap.js

```diff
@@ -0,0 +1,59 @@
+import React, { useCallback, useEffect, useMemo } from 'react';
```

Comment:
[overflow-wrap](issue: add copyright notice)

Reply:
question: what does `UsersStoryMap` mean? How does it relate to / why does it wrap `StoryMap`?

Reply:
@paulschreiber `UserStoryMap` will render a story map owned by a user, `StoryMap` doesn't know from where the configuration for the story map comes from it just renders a story map configuration, that is why it wraps it to provide the context for the specific story map config owned by a user